kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/pom.xml

```diff
@@ -268,6 +268,7 @@
           <configuration>
             <systemPropertyVariables>
               <!-- Values are supplied by specific profiles -->
+              <org.kie.server.bypass.auth.user>true</org.kie.server.bypass.auth.user>
```

Comment:
Not sure why the property is specified in the cargo plugin?

Reply:
it is requires for the server to enable by pass user. Otherwise does not work.